Applications/YOLO/jni/yolo_v2_loss.cpp

```diff
@@ -178,11 +178,9 @@ calc_iou(nntrainer::Tensor &bbox1_x1, nntrainer::Tensor &bbox1_y1,
   if (type_intersection_width == ml::train::TensorDim::DataType::FP32) {
     intersection_width.apply_i<float>(nntrainer::ActiFunc::relu<float>);
   } else if (type_intersection_width == ml::train::TensorDim::DataType::FP16) {
-#ifdef ENABLE_FP16
+    if (!is_fp16_enabled())
+      throw std::runtime_error("Not supported data type: fp16");
```

Comment:
or maybe we can remove these checks and let it be cheked at apply_i() or its underlying function.